hack/benchmark-go.sh

```diff
@@ -20,4 +20,8 @@ set -o pipefail
 
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 
-KUBE_COVER="" KUBE_RACE=" " "${KUBE_ROOT}/hack/test-go.sh" -- -test.run="^X" -benchtime=1s -bench=. -benchmem $@
+make test \
+    WHAT="$*"
```

Comment:
The script previously passed `$@` to test-go.sh, this passes the same args to `make`